client/src/components/Suggestions.jsx

```diff
@@ -7,9 +7,9 @@ import styles from './style.css.js';
 class Suggestions extends React.Component {
   constructor(props) {
     super(props);
+    this.scroller = React.createRef();
```

Comment:
Interesting... what is the Ref used for? Do you need it to make the scroll work?

Reply:
Yes, I found it on google. There's a scrollLeft method in js normally but since we shouldn't interact with the real DOM, using ref can solve the issue.

https://reactjs.org/docs/refs-and-the-dom.html - Refs provide a way to access DOM nodes or React elements created in the render method.

Reply:
fyi the scroll number should change according to screen size, which I will implement later

Reply:
Cool, thanks for the link. I thought abut using Refs for my component, but I found a way that didn't need one. 
I think everything looks good so I'll merge it.